index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <!-- HEADER SECTION-->
+    <header class="header__section">
+        <nav class="navbar__section">
+            <h2 class="navbar__logo">Logo</h2>
+            <i class="navbar__menu fa-solid fa-bars"></i>
+            <div class="navbar__links__container">
+                <a href="#" class="navbar__links">Portfolio</a>
+                <a href="#" class="navbar__links">About</a>
+                <a href="#" class="navbar__links">Contact</a>
+            </div>
+        </nav>
+    </header>
+    <!-- END -->
+
+    <!-- MAIN SECTION -->
+    <main class="main__section">
+        <div class="main__container">
+            <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
+            <p class="main__paragraph">
+                I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <spam class="main__spam">
+                <a href="#" class="main__callToAction">LET'S CONNECT</a>
+            </spam>
+
+            <div class="main__socialMedia">
+                <a href="" class="main__socialMedia__links">
+                    <i class="fa-brands fa-twitter"></i>
+                </a>
+                <a href="" class="main__socialMedia__links">
+                    <i class="fa-brands fa-linkedin-in"></i>
+                </a>
+                <a href="" class="main__socialMedia__links">
+                    <i class="fa-brands fa-github"></i>
+                </a>
+                <a href="" class="main__socialMedia__links">
+                    <i class="fa-brands fa-instagram"></i>
+                </a>
+            </div>
+        </div>
```

Comment:
As mentioned above, kindly I suggest you use a `ul` tag for the list of icons to group them, and `li` per icon, and each icon has a link (`a` tag) to follow HTML good semantics. 😄